### PR TITLE
Fixed post vars

### PR DESCRIPTION
**Pull request by anjeylink**
https://github.com/t3kit/t3kit/issues/92

### CODE REVIEW

File: Classes/Hook/DataHandlerHook.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+use TYPO3\CMS\Core\Utility\MathUtility;
+
+/**
+ * Class DataHandlerHook
+ * @package T3kit\T3kitExtensionTools\Hook
+ */
+class DataHandlerHook
+{
+    /**
+     * If ab child page was created save its uid
```
Comment (MattiasNilsson) [inline Classes/Hook/DataHandlerHook.php]: ab should be a

File: ext_emconf.php [hunk 2]
```diff
@@ -32,17 +32,16 @@
 	'author_company' => '',
 	'CGLcompliance' => '',
 	'CGLcompliance_note' => '',
-	'constraints' => array(
-		'depends' => array(
+	'constraints' => [
+		'depends' => [
 			'php'    => '5.5.0-7.0.99',
 			'typo3'  => '7.5.0-7.6.99',
```
Comment (MattiasNilsson) [inline ext_emconf.php:38]: Is this right dependency?

File: Classes/Utility/HelperUtility.php [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+    /**
+     * Get extension configuration
+     *
+     * @return array|mixed
+     */
+    public static function getExtConf()
+    {
+        if (self::$extConf === null) {
+            self::$extConf = @unserialize((string)$GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['t3kit_extension_tools']);
```
Comment (MattiasNilsson) [inline Classes/Utility/HelperUtility.php]: Why is this suppressed?

**Comment by MattiasNilsson** [comment]
Looks good, just some smaller adjustments needed.

**Comment by anjeylink** [comment]
i fixed wrong comment and suppression of "unserialize". @MattiasNilsson  about dependency, shall we update it to typo3 8  ?

**Comment by dmh** [comment]
@jozefspisiak or @MattiasNilsson

File: Classes/Utility/HelperUtility.php [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+    /**
+     * Get extension configuration
+     *
+     * @return array|mixed
+     */
+    public static function getExtConf()
+    {
+        if (self::$extConf === null) {
+            self::$extConf = unserialize((string)$GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['t3kit_extension_tools']);
```
Comment (MattiasNilsson) [inline Classes/Utility/HelperUtility.php]: Check that this is set before doing unserialize: $GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['t3kit_extension_tools']

File: ext_conf_template.txt [hunk 5]
```diff
@@ -1,2 +1,8 @@
 # cat=basic/enable/100; type=string; label=LLL:EXT:t3kit_extension_tools/Resources/Private/Language/locallang_db.xlf:extmng.additionalChars
-additionalChars =
\ No newline at end of file
+additionalChars =
```
Comment (MattiasNilsson) [inline ext_conf_template.txt]: Please use full naming: additionalCharacters

File: ext_conf_template.txt [hunk 6]
```diff
@@ -1,2 +1,8 @@
...
+
+# cat=basic/enable/110; type=string; label=LLL:EXT:t3kit_extension_tools/Resources/Private/Language/locallang_db.xlf:extmng.fixedPostVarsConfFile
+fixedPostVarsConfFile = typo3conf/ext/t3kit_extension_tools/Configuration/Realurl/predefined_fixedPostVars_conf.php
```
Comment (MattiasNilsson) [inline ext_conf_template.txt]: fixedPostVarsConfFile => fixedPostVarsConfigurationfFile

File: ext_localconf.php [hunk 7]
```diff
@@ -4,6 +4,27 @@
 //
 $init = function ($_EXTKEY) {
     $GLOBALS['TYPO3_CONF_VARS']['SYS']['Objects'][\DmitryDulepov\Realurl\Utility::class]['className'] = \T3kit\T3kitExtensionTools\Xclass\Realurl\Utility::class;
+
+
+    # Register data handler hook
```
Comment (MattiasNilsson) [inline ext_localconf.php]: Replace the usage of # comment to // instead so we all use the same syntax.

**Review by MattiasNilsson — CHANGES_REQUESTED** [review]
This looks good, just some smaller adjustments. This PR should also be marked as breaking change because of the new database column.

**Comment by MattiasNilsson** [comment]
Ping @anjeylink

**Comment by MattiasNilsson** [comment]
Nice!

**Pull request merged**